worlds/sc2/docs/setup_en.md

```diff
@@ -1,110 +1,166 @@
+   If you do not own certain campaigns, you only need to exclude them in the configuration file of your world.
 - [The most recent Archipelago release](https://github.com/ArchipelagoMW/Archipelago/releases)
 
 ## How do I install this randomizer?
 
-1. Install StarCraft 2 and Archipelago using the links above. The StarCraft 2 Archipelago client is downloaded by the Archipelago installer.
+1. Install StarCraft 2 and Archipelago using the links above. The StarCraft 2 Archipelago client is downloaded by the 
+Archipelago installer.
    - Linux users should also follow the instructions found at the bottom of this page 
      (["Running in Linux"](#running-in-linux)).
 2. Run ArchipelagoStarcraft2Client.exe.
-   - macOS users should instead follow the instructions found at ["Running in macOS"](#running-in-macos) for this step only.
-3. Type the command `/download_data`. This will automatically install the Maps and Data files from the third link above.
+   - macOS users should instead follow the instructions found at ["Running in macOS"](#running-in-macos) for this step 
+   only.
+3. Type the command `/download_data`. 
+This will automatically install the Maps and Data files needed to play StarCraft 2 Archipelago.
 
 ## Where do I get a config file (aka "YAML") for this game?
 
-Yaml files are configuration files that tell Archipelago how you'd like your game to be randomized, even if you're only using default options.
+Yaml files are configuration files that tell Archipelago how you'd like your game to be randomized, even if you're only 
+using default options.
 When you're setting up a multiworld, every world needs its own yaml file.
 
 There are three basic ways to get a yaml:
-* You can go to the [Player Options](/games/Starcraft%202/player-options) page, set your options in the GUI, and export the yaml.
-* You can generate a template, either by downloading it from the [Player Options](/games/Starcraft%202/player-options) page or by generating it from the Launcher (ArchipelagoLauncher.exe). The template includes descriptions of each option, you just have to edit it in your text editor of choice.
+* You can go to the [Player Options](/games/Starcraft%202/player-options) page, set your options in the GUI, and export 
+the yaml.
+* You can generate a template, either by downloading it from the [Player Options](/games/Starcraft%202/player-options) 
+page or by generating it from the Launcher (`ArchipelagoLauncher.exe`). 
+The template includes descriptions of each option, you just have to edit it in your text editor of choice.
 * You can ask someone else to share their yaml to use it for yourself or adjust it as you wish.
 
 Remember the name you enter in the options page or in the yaml file, you'll need it to connect later!
 
+The *Player options* page does not allow you to set some advanced options, such as the exclusion of certain units or 
+their upgrades. 
+You can use the [*Weighted Options*](/weighted-options) page to access these settings.
```

Comment:
This is no longer necessary as of 0.5.0

Reply:
Removed in both English and French versions